platform/app/cypress/support/commands.js

```diff
@@ -69,6 +69,18 @@ Cypress.Commands.add(
   }
 );
 
+Cypress.Commands.add('initViewer', (StudyInstanceUID, other = {}) => {
+  const { mode = '/basic-test', minimumThumbnails = 1, params = '' } = other;
+  cy.openStudyInViewer(StudyInstanceUID, params, mode);
+  cy.waitDicomImage();
+  // Very short wait to ensure pending updates are handled
+  cy.wait(25);
```

Comment:
we can go much higher, i don't think 100ms would hurt either

Reply:
It isn't needed - it only needs a very short wait to ensure the next render has completed.  The waitDicomImage is pretty good now about not returning until it is actually done.

Reply:
Making it higher slows the tests down quite a bit.